src/OpenTelemetry.Extensions.Hosting/README.md

```diff
@@ -111,6 +80,25 @@ var app = appBuilder.Build();
 app.Run();
 ```
 
+## Migrating from pre-release versions of OpenTelemetry.Extensions.Hosting
+
+Pre-release versions (all versions prior to 1.4.0) of
+`OpenTelemetry.Extenions.Hosting` contained signal-specific methods for
+configuring tracing and metrics:
+
+* `AddOpenTelemetryTracing`: Configure OpenTelemetry and register an
+  [IHostedService](https://learn.microsoft.com/dotnet/api/microsoft.extensions.hosting.ihostedservice)
+  to automatically start tracing services in the supplied
+  [IServiceCollection](https://learn.microsoft.com/dotnet/api/microsoft.extensions.dependencyinjection.iservicecollection).
+
+* `AddOpenTelemetryMetrics`: Configure OpenTelemetry and register an
+  [IHostedService](https://learn.microsoft.com/dotnet/api/microsoft.extensions.hosting.ihostedservice)
+  to automatically start metric services in the supplied
+  [IServiceCollection](https://learn.microsoft.com/dotnet/api/microsoft.extensions.dependencyinjection.iservicecollection).
+
+These methods were marked obsolete and later removed. You should migrate your
+code to the new `AddOpenTelemetry` method documented above.
```

Comment:
i wonder if we should give an example startup code for this? or the PR which did this for our example one? or maybe link to old and new version of our example app?

Reply:
We could link to https://github.com/open-telemetry/opentelemetry-dotnet/pull/4071 where there are some migration steps.

Reply:
> We could link to #4071 where there are some migration steps.

oops didn't see this comment before I updated it. I just added links to the old (1.3.2 version) and new (main). What do you all think?